tests/network/bond/test_bond_modes.py

```
@@ -65,17 +68,23 @@ def create_vm(namespace, nad, node_selector, unprivileged_client):
+    hosts_common_available_ports,
     worker_node1,
 ):
     """
     Create BOND if setup support BOND
     """
+    if len(hosts_common_available_ports) < 2:
+        LOGGER.error(
+            f"Insufficient common NICs across worker nodes: found {len(hosts_common_available_ports)} "
+            f"({hosts_common_available_ports})"
+        )
+        raise AssertionError("At least 2 common NICs are required.")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)